hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbBulkInserterTest.cpp

```diff
@@ -106,7 +108,7 @@ class ServiceHootApiDbBulkInserterTest : public CppUnit::TestFixture
     writer.setUserEmail(userEmail());
     writer.setCopyBulkInsertActivated(true);
 
-    writer.open(ServicesDbTestUtils::getDbModifyUrl().toString());
+    writer.open(ServicesDbTestUtils::getDbModifyUrl(testName).toString());
```

Comment:
The ability to run in parallel resides here in this call to `ServicesDbTestUtils::getDbModifyUrl` by passing in the name of the test, it opens the database to a map by that name, thus making it unique.